pkg/controllers/deprovisioning/consolidation.go

```diff
@@ -190,7 +195,7 @@ func (c *Consolidation) ValidateCommand(ctx context.Context, nodesToDelete []Can
 	//                    be deleted without producing more than one node
 	// len(newNodes) == 1, as long as the node looks like what we were expecting, this is valid
 	if len(newNodes) == 0 {
-		if cmd.replacementNode == nil {
+		if len(cmd.replacementNodes) == 0 || cmd.replacementNodes[0] == nil {
```

Comment:
`cmd.replacementNodes[0] == nil`
How can we get into this case? This seems indicative of a problem elsewhere.